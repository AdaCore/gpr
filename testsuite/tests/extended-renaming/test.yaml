description:
    Test for extended projects and package renaming
driver: build_and_run
project_file: extended_renaming
main: main
