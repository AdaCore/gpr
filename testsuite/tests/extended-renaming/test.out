Project: c         K_STANDARD
 compiler
   driver (Ada) -> toto
Project: b         K_STANDARD
 compiler
   driver (Ada) -> toto
Project: a         K_STANDARD
 compiler
   driver (Ada) -> toto
